### PR TITLE
Add opengradient testnet

### PR DESCRIPTION
**Pull request by khalifaT**
# Description

Add Opengradient Testnet

 # Testing
 
 message was self relayed

### CODE REVIEW

**Comment by JaunePomme** [comment]
Hi,
 check the README, missing the changeset here (minor), 
may minimify the svg as well :https://jakearchibald.github.io/svgomg/
run prettier,
check how the svg is done in other PR cuz it's more simplified,
the explorer url seems incorrect (api, I don't know),
can modifiy the deployer part in the metadata and add your github for instance

**Comment by khalifaT** [comment]
> Hi, check the README, missing the changeset here (minor), may minimify the svg as well :https://jakearchibald.github.io/svgomg/ run prettier, check how the svg is done in other PR cuz it's more simplified, the explorer url seems incorrect (api, I don't know), can modifiy the deployer part in the metadata and add your github for instance

Thank you for the feedback!
I will proceed with the required updates. However, I wanted to clarify the following points:
* The API link appears to be functional. Are there any specific endpoints or parameters you'd like us to verify or include?
* Deployer in Metadata: For the deployer field, we have referenced the following  examples ([Example 1](https://github.com/hyperlane-xyz/hyperlane-registry/blob/d50740edba1898d3414fac1a675c6b1647c24639/chains/sketchpad/metadata.yaml#L12),[Example 2](https://github.com/hyperlane-xyz/hyperlane-registry/blob/d50740edba1898d3414fac1a675c6b1647c24639/chains/boba/metadata.yaml#L8)). Is it possible to keep the proposed deployer metadata or I have to mention my github account and name.
Thanks again!

**Comment by khalifaT** [comment]
Any updates on this, @JaunePomme ?
Please let me know if there's anything else you need from my side.

File: chains/opengradienttestnet/logo.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
...
+</svg>
```
Comment (paulbalaji) [inline chains/opengradienttestnet/logo.svg:4]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="15.9 16.6 118.1 118.1"><path d="m20 85.3 13.3 13.4c2.8 2.8 6.6 4.4 10.5 4.4h3.8v3.7c0 4 1.5 7.7 4.3 10.5l13.4 13.4c2.6 2.6 6 4 9.7 4s7.1-1.4 9.7-4L98 117.3c2.9-2.8 4.4-6.5 4.4-10.5v-3.7h3.8c4 0 7.7-1.6 10.5-4.4L130 85.3c2.6-2.5 4-6 4-9.7s-1.4-7-4-9.6l-13.3-13.4a14.8 14.8 0 0 0-10.5-4.4h-3.8v-3.7c0-4-1.5-7.7-4.3-10.5L84.7 20.6a13.7 13.7 0 0 0-19.4 0L52 34a14.8 14.8 0 0 0-4.3 10.5v3.7h-3.8c-4 0-7.7 1.6-10.5 4.4L20 66a13.7 13.7 0 0 0 0 19.4Zm73.8 21.5c0 1.7-.7 3.3-1.8 4.4l-13.4 13.4a5 5 0 0 1-7.2 0L58 111.2a6.3 6.3 0 0 1-1.8-4.4v-16c0-1.6.7-3.2 1.8-4.4l4.7-4.7L69 88l-4.7 4.7c-.8.9-1.9 1.4-3 1.7-.8.2-1.4.9-1.4 1.7v5.2c0 1 1 1.9 2 1.7 3.2-.4 6.2-1.9 8.5-4.2L75 94l4.7 4.7c2.8 2.8 6.6 4.4 10.5 4.4h3.6v3.7ZM70.7 75.6a4.4 4.4 0 1 1 1.3 3.1 4.3 4.3 0 0 1-1.3-3ZM98 71c2.7-2.8 4.3-6.6 4.3-10.6V57h3.8c1.6 0 3.2.6 4.4 1.8L124 72c2 2 2 5.3 0 7.2l-13.4 13.4a6.3 6.3 0 0 1-4.4 1.8h-16c-1.7 0-3.3-.6-4.4-1.8L81 88l6.2-6.2 4.7 4.7c.8.9 1.4 2 1.6 3 .2.9 1 1.4 1.7 1.4h5.3c1 0 1.8-.9 1.7-2-.4-3.1-2-6.1-4.2-8.5l-4.7-4.7L98 71ZM56.2 44.5c0-1.7.7-3.3 1.8-4.4l13.4-13.4c2-2 5.2-2 7.2 0L92 40a6.3 6.3 0 0 1 1.8 4.4v16c0 1.6-.7 3.2-1.8 4.4l-4.7 4.7-6.2-6.2 4.7-4.7c.8-.9 1.9-1.4 3-1.7.8-.2 1.4-.9 1.4-1.7v-5.2c0-1-1-1.9-2-1.7-3.2.4-6.2 1.9-8.5 4.2L75 57.3l-4.7-4.7a14.9 14.9 0 0 0-10.5-4.4h-3.6v-3.7ZM26 72l13.4-13.3a6.3 6.3 0 0 1 4.4-1.8h16c1.7 0 3.3.6 4.4 1.8l4.7 4.7-6.2 6.2-4.7-4.7c-.8-.9-1.4-2-1.6-3.1-.2-.8-1-1.4-1.7-1.4h-5.3c-1 0-1.8 1-1.7 2 .4 3.2 2 6.2 4.2 8.6l4.7 4.6-4.6 4.7a14.9 14.9 0 0 0-4.4 10.5v3.6h-3.8c-1.6 0-3.2-.6-4.4-1.8L26 79.2a5 5 0 0 1 0-7.2Z"/></svg>
```

File: chains/opengradienttestnet/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+blockExplorers:
+  - apiUrl: http://3.145.62.2/api
+    family: blockscout
+    name: Opengradient Testnet Explorer
+    url: http://3.145.62.2
+chainId: 2970285607590380
```
Comment (paulbalaji) [inline chains/opengradienttestnet/metadata.yaml:6]: please also add block information such as https://github.com/hyperlane-xyz/hyperlane-registry/blob/aa6e8bf36dbe75ca8b7f4a1ba784e61ae5fd4e58/chains/abstracttestnet/metadata.yaml#L7-L10

**Review by paulbalaji — COMMENTED** [review]
please also run yarn lint:fix and yarn prettier

**Comment by JaunePomme** [comment]
Yes, I think it can be merged now, everything has been taken into account

**Comment by khalifaT** [comment]
Any updates on the PR?
Thanks

**Comment by JaunePomme** [comment]
They will review it, don't worry just gotta be patient, team is building hard !

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**